includes/validation/class-amp-validation-manager.php

```diff
@@ -168,24 +168,25 @@ class AMP_Validation_Manager {
 	protected static $stylesheet_slug;
 
 	/**
-	 * Add the actions.
+	 * Initialize.
 	 *
-	 * @param array $args {
-	 *     Args.
-	 *
-	 *     @type bool $should_locate_sources Whether to locate sources.
-	 * }
 	 * @return void
 	 */
-	public static function init( $args = [] ) {
-		$args = array_merge(
-			[
-				'should_locate_sources' => self::should_validate_response(),
-			],
-			$args
-		);
+	public static function init() {
+		$should_validate_response = self::should_validate_response();
```

Comment:
True, but a reason against this is that `\AMP_Validation_Manager::$is_validate_request` is defined as a `bool`. It would seem a bit weird to save a `WP_Error` instance into it.